Resources/ModelingAndBinding-Starter/Podfile

```diff
@@ -0,0 +1,11 @@
+# Uncomment the next line to define a global platform for your project
+platform :ios, '10.0'
+
+use_frameworks!
+
+pod 'IGListKit', :git => 'https://github.com/Instagram/IGListKit.git', :branch => 'master'
```

Comment:
this needs to reference the local spec. `../../IGListKit.podspec`

otherwise, we've duped all source into this directory 😅 

why not put this stuff in the existing `Examples/` dir?